src/interp.cc

```diff
@@ -1047,6 +1070,12 @@ ValueTypeRep<R> IntEqz(ValueTypeRep<T> v_rep) {
   return ToRep(v_rep == 0);
 }
 
+// i{32,64}.neg
```

Comment:
nit: this isn't a real instruction, so it's better to just remove this comment.

Reply:
Done.